core/src/Kokkos_View.hpp

```diff
@@ -911,7 +918,7 @@ class View : public ViewTraits<DataType, Properties...> {
                                (0 == Rank)),
                               reference_type>::type
       operator()(Args... args) const {
-    KOKKOS_IMPL_VIEW_OPERATOR_VERIFY((m_track, m_map, args...))
+    KOKKOS_IMPL_VIEW_OPERATOR_VERIFY((m_track.m_tracker, m_map, args...))
```

Comment:
Can we just change the definition of the function associated with `KOKKOS_IMPL_VIEW_OPERATOR_VERIFY` rather than touching all of the call sites? Or at least add an overload that does this?

Reply:
+1 on making it so we don't need this awkward `m_track -> m_track.m_tracker` change everywhere.

Reply:
You could also think about an implicit conversion operator. It really only wraps this one variable.